### PR TITLE
Hazy Food Tweaks Patch

### PR DESCRIPTION
**Pull request by HazelHyena**
# Description

A quick update to food tweaks, fixing things on the backend for mappers and removing duplicate Meme Garbage.

# Changelog

:cl:
- removed: Every SS14 meme recipe that didn't fit N14 (the bearger persists, for example)
- added: flour bags contain razorgrain flour and cornmeal bags contain maize flour. This has been tweaked to prevent conflicts with SS14 recipes upstream - all N14 dough recipes SHOULD require these ingredients in place of the upstream reagents. (this should allow pizzas to work)
- Changed the naming structure for all recipes to make spawning things easier for admins and mappers.
- removed: a few recipes were throwing up errors and not working. They're out for now.
- We have N14 egg cartons now which contain N14 eggs. Until chickens are properly sorted out, this will allow recipe usage in the interim!

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml
Comment (HazelHyena) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/soup.yml:1]: oops this was the upstream one. I'll fix that now

File: Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_meal.yml
Comment (HazelHyena) [inline Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/food_meal.yml:1]: also getting rid of this, pep fixed this (why did it add this text from earlier? bleh)

**Comment by HazelHyena** [comment]
Don't merge this just yet, please! Give me a shout on discord, I need getting rid of robust toolbox clientside bugs so I can test the pizza recipes :)

**Comment by Partmedia** [comment]
I'm reverting this PR because this causes the Sunnyvale map to fail to load. Please feel free to re-PR after you've taken a look at the issue and fixed it.

**Pull request merged**